calendar_resource/README.rst

```diff
@@ -10,11 +10,26 @@ This module adds new features to facilitate resource management with meetings:
 
 * To attach resources to a meeting, go to `Calendar` => Click on a meeting => `Edit` =>
   `Edit` => add any related resources.
-* To edit or add a resource, go to `Settings` => `Resource` => `Resources`.
+* To edit or add a resource, go to `Settings` => `Resource` => `Resources`. Developer mode
+  has to be enabled for this menu to show.
```

Comment:
@dreispt You mean adding to the Calendar page directly? I see there's some space on the right there, but I don't think I've seen any apps actually populate that area before. Perhaps there are and I've just never seen any. Is that where you're thinking?
![fireshot capture 8 - meetings - odoo_ - http___192 168 1 133_8069_web](https://user-images.githubusercontent.com/11901801/29191084-584098c8-7dd1-11e7-9374-9f581435553e.png)